frontend/src/bundles/studio/components/timeline/timecursor.tsx

```diff
@@ -0,0 +1,130 @@
+import { useTimelineContext } from 'dnd-timeline';
+
+import { Box } from '~/bundles/common/components/components.js';
+import {
+    useCallback,
+    useLayoutEffect,
+    useRef as useReference,
+    useState,
+} from '~/bundles/common/hooks/hooks.js';
+
+type TimeCursorProperties = {
+    interval?: number;
+};
+
+const TimeCursor = (properties: TimeCursorProperties): JSX.Element | null => {
```

Comment:
Same here

Reply:
Also, why do you need `JSX.Element | null` if `null `condition is never met here?

Reply:
Sorry, my first approach was to make the whole component collapsible but after I changed the idea I supposed I didnt delete all the traces. Ill check all files for other errors like this. Thank you.